### ISSUE TITLE
Small autocompletion bug

### ISSUE THREAD

**Issue by pianoslum** [protz/GMail-Conversation-View#934]
STB:
* Reply to a message in conversation mode
* Click on the "to"-address-bar to add a new recipient
* Type "asdf" - a pop-"down" with suggestions (the only one being "asdf") will open
* Click on the white part of that pop-down, not the highlighted part with the text - it disappears and you can continue typing
* Write "<Space>qwer"
* Now click on the highlighted part

Timestamp: 31.03.2015 18:34:37
Error: TypeError: item is null
Source File: chrome://conversations/content/js/jquery.tokeninput.js
Line: 363

**Comment by protz** [comment]
Do you know of a good plugin that would achieve similar functionality 
but is well-maintained, takes asynchronous completion, allows moving 
tokens from one field to another, and doesn't have as many bugs?

I already removed jquery-tmpl in favor of handlebars this weekend, I 
guess I could upgrade the completion plugin too while I'm at it...

**Comment by pianoslum** [comment]
No, I'm sorry ;-)
But I'd be in favour of using an existing lib rather than writing/maintaining everything on our own!

**Comment by protz** [comment]
Yes, definitely. This was already something I imported years ago, but I 
need to find out if something better exists now...

**Comment by protz** [comment]
http://sliptree.github.io/bootstrap-tokenfield/ seems to fit the bill and would probably have less UI bugs than the current implementation; it's modern and well-maintained, and it shouldn't be more than an hour's work to port our current code to the new plugin. That being said, you still can't drag&drop tokens between fields.

**Comment by protz** [comment]
For the record, bootstrap-tokenfield doesn't seem to support asynchronous completion, so a bit of hacking would be needed first.

**Comment by msprotz** [comment]
https://github.com/sliptree/bootstrap-tokenfield/pull/236 pull request for drag&drop there. Seems closer to usable for us now.

**Comment by Standard8** [comment]
Currently we don't have autocompletion, if/when we add that we can re-check for issues.
